saskatoon/harvest/signals.py

```diff
@@ -41,7 +43,10 @@ def changed_by(sender, instance, **kwargs):
 def notify_pending_status_update(sender, instance, **kwargs):
     # Send email only if pending status is removed
     if instance.id:
-        original_instance = sender.objects.get(id=instance.id)
+        try:
+            original_instance = sender.objects.get(id=instance.id)
```

Comment:
```suggestion
            # this might raise DoesNotExist error while loading the fixtures
            # see https://github.com/LesFruitsDefendus/saskatoon-ng/issues/247 for more informations.
            original_instance = sender.objects.get(id=instance.id)
```